app/components/ProblemsTable.tsx

```diff
@@ -44,7 +44,17 @@ export function ProblemsTable({ problems }: ProblemsTable) {
                   {problem.title}
                 </Link>
               </td>
-              <td className="p-2">{problem.difficulty}</td>
+              <td
+                className={`px-2 ${
+                  problem.difficulty === 'easy'
+                    ? 'text-green-500'
+                    : problem.difficulty === 'medium'
+                      ? 'text-yellow-500'
+                      : 'text-red-500'
+                }`}
+              >
+                {problem.difficulty}
+              </td>
```

Comment:
To these cases, you can use the `cn` util function.
See a example here: https://github.com/ProgramadoresSemPatria/gama-module-code-breakers/blob/87f542fdacd9d8a7bcdd53ec48981a6a97ab8f84/app/components/Sidebar.tsx#L19-L23